gcov_support.sh

```diff
@@ -0,0 +1,456 @@
+#!/bin/bash
```

Comment:
i think this whole file should go to swss repo and better integrate with swss vs test.